osf/models/subject.py

```diff
@@ -1,25 +1,33 @@
 # -*- coding: utf-8 -*-
 from django.db import models
+from django.db.models import Q
 from django.core.exceptions import ValidationError
 from django.utils.functional import cached_property
+from include import IncludeQuerySet
 
 from website.util import api_v2_url
 
-from osf.models.base import BaseModel, ObjectIDMixin
+from osf.models.base import BaseModel, MODMCompatibilityQuerySet, ObjectIDMixin
 from osf.models.preprint_provider import PreprintProvider
 from osf.models.validators import validate_subject_hierarchy_length, validate_subject_provider_mapping
 
+class SubjectQuerySet(MODMCompatibilityQuerySet, IncludeQuerySet):
+    def include_children(self):
+        return (self | Subject.objects.filter(Q(parent__in=self) | Q(parent__parent__in=self)))
 
 class Subject(ObjectIDMixin, BaseModel):
     """A subject discipline that may be attached to a preprint."""
-    modm_model_path = 'website.project.taxonomies.Subject'
-    modm_query = None
 
-    text = models.CharField(null=False, max_length=256, unique=True)  # max length on prod: 73
+    text = models.CharField(null=False, max_length=256)  # max length on prod: 73
     parent = models.ForeignKey('self', related_name='children', null=True, blank=True, on_delete=models.SET_NULL, validators=[validate_subject_hierarchy_length])
     bepress_subject = models.ForeignKey('self', related_name='aliases', null=True, blank=True, on_delete=models.deletion.CASCADE)
     provider = models.ForeignKey(PreprintProvider, related_name='subjects', on_delete=models.deletion.CASCADE)
 
+    objects = SubjectQuerySet.as_manager()
+
```

Comment:
Resolved in https://github.com/CenterForOpenScience/osf.io/pull/7163/commits/60c380ee177c69f6da2c166e5a090d760664e9e1